### ISSUE TITLE
How to solve lr = 0 after training 5 epochs

### ISSUE THREAD

**Issue by kkkris7** [clovaai/bros#11]
Thank you for the amazing work! 
I am training the model with a customized dataset. However, I just noticed after 5 epochs training, the learning rate came to 0 which makes model hard to learn. Could you please point me to the learning rate strategy of BROS and may I know how to change it according to my case? Thanks!

TRAIN [epoch: 0/50] || train_loss: 460.69653 || lr: 4e-05 || time: 193.6 secs.
precision: 0.9080, recall: 0.9023, f1: 0.9052
TRAIN [epoch: 1/50] || train_loss: 129.8502 || lr: 3e-05 || time: 198.6 secs.
precision: 0.9374, recall: 0.9184, f1: 0.9278
TRAIN [epoch: 2/50] || train_loss: 75.951 || lr: 3e-05 || time: 198.0 secs.
precision: 0.9293, recall: 0.9183, f1: 0.9237
TRAIN [epoch: 3/50] || train_loss: 46.87292 || lr: 2e-05 || time: 197.8 secs.
precision: 0.9442, recall: 0.9391, f1: 0.9416
TRAIN [epoch: 4/50] || train_loss: 28.64673 || lr: 1e-05 || time: 197.7 secs.
precision: 0.9444, recall: 0.9392, f1: 0.9418
TRAIN [epoch: 5/50] || train_loss: 16.82515 || lr: 0.0 || time: 197.6 secs.

**Comment by kkkris7** [comment]
problem solved! Thanks

**Comment by f-izzat** [comment]
How was the problem solved?
